validation/src/test/scala/hmda/validation/rules/lar/macro/Q016Spec.scala

```diff
@@ -13,12 +13,17 @@ class Q016Spec extends MacroSpec {
   val sampleSize = testLars.size
   val sampleSizeTarget = (testLars.size * multiplier).toInt
   def underCap(lar: LoanApplicationRegister) = {
-    val underCapLoan = lar.loan.copy(amount = incomeCap - 1)
-    lar.copy(loan = underCapLoan)
+    val underCapApplicant = lar.applicant.copy(income = (incomeCap - 1).toString)
+    lar.copy(applicant = underCapApplicant)
   }
   def overCap(lar: LoanApplicationRegister) = {
-    val overCapLoan = lar.loan.copy(amount = incomeCap + 1)
-    lar.copy(loan = overCapLoan)
+    val overCapApplicant = lar.applicant.copy(income = (incomeCap + 1).toString)
```

Comment:
We should use `incomeCap` here, not `incomeCap + 1`, so that the spec covers the edge case.